test/manipulation.test.js

```diff
@@ -215,18 +203,6 @@ describe('manipulation', function() {
         }).catch(done);
     });
 
-    it('should work when called with no data and callback', function(done) {
-      Person.afterCreate = function(next) {
-        this.should.be.an.instanceOf(Person);
-        should.not.exist(this.name);
-        should.exist(this.id);
-        Person.afterCreate = null;
-        next();
-        setTimeout(done, 30);
-      };
-      Person.create();
```

Comment:
I am not sure that we have a unit-test for this invocation, could you please check?